release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2745,6 +2745,40 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+//4.15.36
+[id="ocp-4-15-36_{context}"]
+=== RHSA-2024:7594 - {product-title} 4.15.36 bug fix and security update
+
+Issued: 09 October 2024
+
+{product-title} release 4.15.36, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:7594[RHSA-2024:7594] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHBA-2024:7597[RHBA-2024:7597] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.36 --pullspecs
+----
+
+[id="ocp-4-15-36-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, if a connection on port 9637 for Windows nodes was refused, the Kubelet Service Monitor threw a `target down` alert because CRIO doesn't run on Windows nodes. With this release, Windows nodes are excluded from the Kubelet Service Monitor. (link:https://issues.redhat.com/browse/OCPBUGS-42586[*OCPBUGS-42586*])
+
+* Previously, a change in the ordering of the `TextInput` parameters for PatternFly v4 and v5 caused the `until` field to be improperly filled and you could not edit it. With this release, the `until` field is editable so you can input the correct information. (link:https://issues.redhat.com/browse/OCPBUGS-42834[*OCPBUGS-42384*])
+
+* Previously, when the Node Tuning Operator (NTO) was configured using PerformanceProfiles, it created the `ocp-tuned-one-shot systemd` service. The service ran before Kubelet and blocked the service execution. The `systemd` service invoked Podman, which used the NTO image. If the NTO image was not present, Podman tried to fetch the image. 
```

Comment:
I think this is just "performance profiles", e.g. https://docs.openshift.com/container-platform/4.17/scalability_and_performance/low_latency_tuning/cnf-tuning-low-latency-nodes-with-perf-profile.html#cnf-about-the-profile-creator-tool_cnf-low-latency-perf-profile
```suggestion
* Previously, when the Node Tuning Operator (NTO) was configured using performance profiles, it created the `ocp-tuned-one-shot systemd` service. The service ran before Kubelet and blocked the service execution. The `systemd` service invoked Podman, which used the NTO image. If the NTO image was not present, Podman tried to fetch the image. 
```